test/hotspot/jtreg/compiler/vectorapi/VectorUnsignedMinMaxOperationsTest.java

```diff
@@ -0,0 +1,472 @@
+            int actual = int_out[i];
+            int expected = VectorMath.minUnsigned(int_in1[i], int_in1[i]);
+            if (actual != expected) {
+                throw new AssertionError("Result Mismatch : actual (" +  actual + ") !=  expected (" + expected  + ")");
+            }
+        }
+    }
+
+    @Test
+    @IR(counts = {IRNode.UMAX_VI, " 0 "}, applyIfCPUFeature = {"avx", "true"})
+    @Warmup(value = 10000)
+    public void umax_ir_transform1() {
+        for (int i = 0; i < COUNT; i += ispec.length()) {
+            IntVector.fromArray(ispec, int_in1, i)
+                     .lanewise(VectorOperators.UMAX,
+                               IntVector.fromArray(ispec, int_in1, i))
+                     .intoArray(int_out, i);
+        }
+    }
+
+    @Check(test = "umax_ir_transform1", when = CheckAt.COMPILED)
+    public void umax_ir_transform1_verify() {
+        for (int i = 0; i < COUNT; i++) {
+            int actual = int_out[i];
+            int expected = VectorMath.maxUnsigned(int_in1[i], int_in1[i]);
+            if (actual != expected) {
+                throw new AssertionError("Result Mismatch : actual (" +  actual + ") !=  expected (" + expected  + ")");
+            }
+        }
+    }
+
+    @Test
+    @IR(counts = {IRNode.UMAX_VI, " 0 ", IRNode.UMIN_VI, " >0 "}, applyIfCPUFeature = {"avx", "true"})
+    @Warmup(value = 10000)
+    public void umin_max_ir_transform1() {
+        for (int i = 0; i < COUNT; i += ispec.length()) {
+            IntVector vec1 = IntVector.fromArray(ispec, int_in1, i);
+            IntVector vec2 = IntVector.fromArray(ispec, int_in2, i);
+            // UMinV (UMinV vec1, vec2) (UMaxV vec1, vec2) => UMinV vec1 vec2
+            vec1.lanewise(VectorOperators.UMIN, vec2)
+                .lanewise(VectorOperators.UMIN,
+                          vec1.lanewise(VectorOperators.UMAX, vec2))
+                .intoArray(int_out, i);
+        }
+    }
+
+    @Check(test = "umin_max_ir_transform1", when = CheckAt.COMPILED)
+    public void umin_max_ir_transform1_verify() {
+        for (int i = 0; i < COUNT; i++) {
+            int actual = int_out[i];
+            int expected = VectorMath.minUnsigned(VectorMath.minUnsigned(int_in1[i], int_in2[i]),
+                                                  VectorMath.maxUnsigned(int_in1[i], int_in2[i]));
+            if (actual != expected) {
+                throw new AssertionError("Result Mismatch : actual (" +  actual + ") !=  expected (" + expected  + ")");
+            }
+        }
+    }
+
+    @Test
+    @IR(counts = {IRNode.UMIN_VI, " 0 ", IRNode.UMAX_VI, " >0 "}, applyIfCPUFeature = {"avx", "true"})
+    @Warmup(value = 10000)
+    public void umin_max_ir_transform2() {
+        for (int i = 0; i < COUNT; i += ispec.length()) {
+            IntVector vec1 = IntVector.fromArray(ispec, int_in1, i);
+            IntVector vec2 = IntVector.fromArray(ispec, int_in2, i);
+            // UMaxV (UMinV vec2, vec1) (UMaxV vec1, vec2) => UMinV vec1 vec2
```

Comment:
The above comment should be:
// UMaxV (UMinV vec2, vec1) (UMaxV vec1, vec2) => UMaxV vec1 vec2